source/opt/type_manager.cpp

```diff
@@ -59,7 +59,9 @@ std::pair<Type*, std::unique_ptr<Pointer>> TypeManager::GetTypeAndPointerType(
 
 uint32_t TypeManager::GetId(const Type* type) const {
   auto iter = type_to_id_.find(type);
-  if (iter != type_to_id_.end()) return (*iter).second;
+  if (iter != type_to_id_.end()) {
+    return (*iter).second;
+  }
```

Comment:
In our offline chat you mentioned that you'd want to get rid of this NOP change.  I don't mind it, actually.  I prefer every control construct to have braces even for one-liners.